code/modules/clothing/under/syndicate.dm

```diff
@@ -267,7 +267,7 @@
 	name = "inteq corpsman skirtleneck"
 	desc = "A sterile white turtleneck with a free flowing black skirt, it is emblazoned with the lettering 'IRMG' on the shoulder. For the IRMG's support division Corpsmen."
 	icon_state = "inteqmed_skirt"
-	supports_variations = KEPORI_VARIATION | DIGITIGRADE_VARIATION_NO_NEW_ICON
+	supports_variations = KEPORI_VARIATION | VOX_VARIATION | DIGITIGRADE_VARIATION_NO_NEW_ICON
```

Comment:
I think this got mixed in.

Reply:
Accidental addition, but I suppose it's a fix part of the PR now. Resolved without a comment, my bad.